css/style.css

```diff
@@ -0,0 +1,28 @@
+.login-form,
+.logout-form {
+	width: 340px;
+	margin: 50px auto;
+}
+.login-form form,
+.logout-form form {
+	margin-bottom: 15px;
+	background: #f7f7f7;
+	box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.3);
+	padding: 30px;
+}
+.login-form h2,
+.logout-form h2 {
+	margin: 0 0 15px;
+}
+.form-control,
+.btn {
+	min-height: 38px;
+	border-radius: 2px;
+}
+.btn {
+	font-size: 15px;
```

Comment:
I'm pretty sure this can be done via bootsrap class.

Reply:
All right, OK, I got it.